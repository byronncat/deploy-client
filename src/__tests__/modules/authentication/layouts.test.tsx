import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

import { LandingLayout } from '@authentication';

describe('authentication layout', () => {
  it('should render correctly when not authenticated', () => {
    const { container } = render(<LandingLayout title="login" />);
    expect(container).toMatchSnapshot();
  });
});
