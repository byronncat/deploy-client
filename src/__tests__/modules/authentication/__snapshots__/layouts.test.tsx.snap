// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`authentication layout should render correctly when not authenticated 1`] = `
<div>
  <div
    class="w-full h-full relative overflow-hidden select-none"
  >
    <span
      class="w-full h-full block absolute z-0"
      style="background-image: url(night-neon.avif); background-position: center center; background-size: cover;"
    />
    <div
      class="text-white w-full md:w-1/2 h-full relative z-10 flex flex-col justify-center items-center duration-500 ease-in-out"
    >
      <img
        alt="logo"
        class="block"
        height="60"
        src="logo.svg"
        width="60"
      />
      <h1
        class="text-neon-glowing-1 mt-2 mb-6 font-monoton text-3xl tracking-widest -skew-y-6"
      >
        bygram
      </h1>
      <span
        class="panel-background w-full h-full absolute top-0 -z-10"
      />
      <span
        class="text-neon-glowing-2 px-4 py-2 m-2 font-sacramento text-4xl text-capitalize capitalize absolute top-0 left-0"
      >
        login
      </span>
      <span
        class="text-neon-glowing-2 px-4 py-2 m-2 bg-slate-950/50 rounded-lg font-sacramento text-4xl text-center text-capitalize capitalize flex flex-col items-center justify-center absolute top-0 left-full"
      >
        welcome back
      </span>
    </div>
  </div>
</div>
`;
